roles/edpm_kernel/meta/argument_specs.yml

```diff
@@ -52,3 +52,7 @@ argument_specs:
         type: dict
         default: {}
         description: Hugepages configuration dictionary.
+      edpm_kernel_enable_ksm:
+        default: false
+        description: ''
```

Comment:
can you add a short description here, pls? :)

Even "Enable KSM" would be fine.

Reply:
good point, more docs never can hurt